conf/st2.dev.conf

```diff
@@ -5,7 +5,6 @@
 host = 0.0.0.0
 port = 9101
 logging = st2api/conf/logging.conf
-serve_webui_files = True
 # allow_origin is required for handling CORS in st2 web UI.
 # allow_origin = http://myhost1.example.com:3000,http://myhost2.example.com:3000
```

Comment:
In production, we probably want nginx to handle CORS. At least he knows what our hostname is beforehand. Ok, I'll add awk replace in st2_deploy, we'll figure out the rest later.